app/views/course/statistics/assessments/main_statistics.json.jbuilder

```diff
@@ -13,7 +13,7 @@ json.submissions @student_submissions_hash.each do |course_user, (submission, an
     json.name name
   end
 
-  if !submission.nil? && submission.workflow_state == 'published' && submission.grader_ids
+  if !submission.nil? && submission.workflow_state.in?(['graded', 'published']) && submission.grader_ids
```

Comment:
double-check if there a `submission.graded?` instead of `submission.workflow_state.in?...`

Reply:
yes there is, however if we switch to that it will be just as unclean as the current one
(the option is between `submission.graded? || submission.published?` and `submission.workflow_state.in?(['graded', 'published'])`)

I personally prefer the second one since it's more concise.

Reply:
What do you mean by concise? Isn't the first-way shorter?
And first way makes use of the the submission attributes instead of relying on hardcoded strings.

Reply:
actually both are just as long.. but if u prefer the first one, I have no problem in changing it.
Also you got the point regarding the hardcoded strings, it makes the code look messier

Reply:
How are both just as long?  
Yes please change, it's not so much about messy but more of avoid magic strings follow existing convention where we can.

Reply:
done